recipes-kernel/linux/linux-fslc/defconfig

```diff
@@ -473,3 +473,4 @@ CONFIG_DEBUG_FS=y
 CONFIG_PROVE_LOCKING=y
 # CONFIG_DEBUG_BUGVERBOSE is not set
 # CONFIG_FTRACE is not set
+CONFIG_KPROBES=y
```

Comment:
Please add the end of the line on the file.

Reply:
Added the missing end of line on the file. Thanks for the review!

Reply:
Squash the commits please.

Reply:
Commits are now squashed as requested